queries/boundaries.jinja2

```diff
@@ -112,27 +112,4 @@ FROM buffered_land
 WHERE
   {{ bounds['polygon']|bbox_filter('the_geom',3857) }}
 
-UNION ALL
```

Comment:
Does removing this UNION prevent export of any boundary line data that is not polygons?